index.js

```diff
@@ -374,24 +376,40 @@ async function runTests(handle) {
 
```

Comment:
the reconciliations liquid content that we provide is not including shared parts.

Function  constructReconciliationText(handle) only provides main and local parts, not shared parts. 

Is not clear in the [documentation](https://live.getsilverfin.com/api_docs#!/templates/postV4FRootFirmIdReconciliationsTest) as well how it should be handled

When I run a test from the CLI which has shared parts I get this error: 
`Liquid error: Liquid error (line 4): Unknown part shared/general_translations include "shared/general_translations"`
While running the same test in the Platform IDE, all tests pass.